docs/reference/getting-started-with-logstash.md

```diff
@@ -62,9 +64,10 @@ OpenJDK 64-Bit Server VM Temurin-17.0.12+7 (build 17.0.12+7, mixed mode)
 On some Linux systems, you may need to have the `LS_JAVA_HOME` environment exported before installing {{ls}}, particularly if you installed Java from a tarball. {{ls}} uses Java during installation to automatically detect your environment and install the correct startup method (SysV init scripts, Upstart, or systemd). If {{ls}} is unable to find the `LS_JAVA_HOME` environment variable during package installation, you may get an error message, and {{ls}} will not start properly.
 
 
-#### Using JDK 17 [jdk17-upgrade]
+#### Update JDK settings when upgrading from {{ls}} 7.11.x (or earlier)[jdk-upgrade]
 
-{{ls}} uses JDK 17 by default, but you need to update settings in `jvm.options` and `log4j2.properties` if you are upgrading from  {{ls}} 7.11.x (or earlier) to 7.12 or later.
+{{ls}} uses JDK 21 by default.
+If you are upgrading from {{ls}} 7.11.x (or earlier), you need to update Java settings in `jvm.options` and `log4j2.properties`.
```

Comment:
@robbavey @mashhurs What do you think about this approach to handling the need to update JVM settings when upgrading from  LS 7.11.x or earlier?  It's simpler and more straightforward... but is it accurate?

Reply:
I'm fine with this, but I would like to add a follow-on to add additional information for users of the elastic integration filter requiring jdk21

Reply:
`elastic_integration` plugin was embedded in 8.13 and it doesn't seem to me this section is relative (7.11.x versions do not include the plugin or cannot run integrations).

If we still would like to include, something like would be suitable: _If your pipelines include `elastic_integration` plugin, checkout plugin [requirements](https://www.elastic.co/guide/en/logstash/current/plugins-filters-elastic_integration.html#plugins-filters-elastic_integration-requirements)_

Reply:
You are right that it doesn't affect users upgrading from `7.x`, but in the context of JDK requirements, the use of the `elastic_integration` filter prevents the use of JDK17, so I would like to see a caveat for there somewhere. 

I think the subtleties of this make it appropriate to discuss in a separate PR

Reply:
> I think the subtleties of this make it appropriate to discuss in a separate PR
Agreed, Rob.  We're in the post-migration cleanup phase, and need to move quickly. What do you need to see in this PR to feel comfortable signing off?

Reply:
LGTM. Let's move on